test/Test/Toml/Gen.hs

```diff
@@ -119,3 +129,33 @@ genToml = do
     kv     <- HashMap.fromList <$> genKeyAnyValueList
     tables <- Gen.list (Range.linear 0 10) genTableHeader
     pure $ TOML kv (fromList tables)
+
+genDay :: MonadGen m => m DateTime
+genDay = do
+    y <- toInteger <$> Gen.int (Range.constant 1968 2019)
+    m <- Gen.int (Range.constant 1 12)
+    d <- Gen.int (Range.constant 1 28)
+    let day = fromGregorian y m d
+    pure $ Day day
+
+genHours :: MonadGen m => m DateTime
+genHours = do
+    secs <- MkFixed <$> Gen.integral (Range.constant 0 61)
+    mins <- Gen.int (Range.constant 0 59)
+    hours <- Gen.int (Range.constant 0 23)
+    pure $ Hours $ TimeOfDay hours mins secs
+
+genLocal :: MonadGen m => m DateTime
+genLocal = do
+    Day day <- genDay
+    Hours time <- genHours
+    let local = LocalTime day time
+    pure $ Local local
+
+genZoned :: MonadGen m => m DateTime
+genZoned = do
+    Local local <- genLocal
+    zMin <- Gen.int (Range.constant 1 720)
```

Comment:
Looks like this generator can take negative minutes. It's totally valid to have negative timezone like `-0800`.